pytest_testrail/plugin.py

```diff
@@ -27,11 +27,13 @@
 
 ADD_RESULTS_URL = 'add_results_for_cases/{}'
 ADD_TESTRUN_URL = 'add_run/{}'
+ADD_PLANENTRY_URL = 'add_plan_entry/{}'
```

Comment:
`ADD_PLAN_ENTRY_URL`

Reply:
Will do, kind of went back and forth on that

Reply:
Done